account_invoice_supplierinfo_update/models/account_move_line.py

```diff
@@ -12,8 +12,9 @@ def _get_supplierinfo(self):
         """Given an invoice line, return the supplierinfo that matches
         with product and supplier, if exist"""
         self.ensure_one()
-        supplierinfos = self.product_id.seller_ids.filtered(
-            lambda seller: seller.partner_id == self.move_id.supplier_partner_id
+        supplierinfos = self.product_id._select_seller(
+            partner_id=self.move_id.supplier_partner_id,
+            quantity=self.quantity,
```

Comment:
"Information is a mass noun. It has no plural form"* so infos looks like informations which would be Frenglish and looks TinyERP. Better change that to supplier_info or supplier_data as you want.

* https://preply.com/en/question/plural-of-information

Reply:
Hi. 

@rvalyi. We already talk about that in another PR. summary of the previous comments : 

what you mention is not an OCA rule. 
- "infos" is used in many OCA repo. (12 occurences in V16, only in the repos I cloned for GRAP project)
- "datas" is used in many OCA repo. (19 occurences in V16, only in the repos I cloned for GRAP project)

So if you want to introduce that new rule, please create a dedicated PR here https://github.com/OCA/odoo-community.org/tree/master/website/Contribution

In the meantime, please avoid to block valid PR.

Thanks !

Reply:
I didn't block, only suggest to use better variable names. When native English speakers read that they feel it's cheap while it's easy to improve.

Reply:
1)
> I didn't block

You set "changes requested". Could you remove it ?

2) as said not the place to create new rules. Please create a dedicated issue. 

Thanks !